README.md

```diff
@@ -162,6 +171,16 @@ This will create a clean environment where is uses the [pip-tools](https://githu
 
 ## Testing
 
+Local development testing:
+
+```bash
+cd path/to/repo
+docker-compose exec api bash
+root@3c5df91778ad:/code# pytest
```

Comment:
Why not `docker-compose exec api pytest`?

Reply:
Edit: it should probably be `$ docker exec` instead of `$ docker-compose exec`

Reply:
That can work as well. This is just an example for how it can be done, in the end people can use as they like. With the above approach I find that it is easier to do follow-ups on test failures, like `pytest --lf` or `pytest --pdb` (less typing :))